version.json

```diff
@@ -1,7 +1,7 @@
 {
-  "version": "15.1-preview5",
+  "version": "15.1-preview",
```

Comment:
I just checked and actually this will be fine.  People will specify `15.1.0-preview-*` and we've only published 1 package as version `15.1.0` and it's number is low enough that it will sort as last.

So we can keep `preview` as the tag unless we want it to be something else